packages/jest-config/src/Defaults.ts

```diff
@@ -71,6 +71,7 @@ const defaultOptions: Config.DefaultOptions = {
   testRegex: [],
   testResultsProcessor: null,
   testRunner: 'jasmine2',
+  testSequencer: '@jest/test-sequencer',
```

Comment:
This value doesn't get fully resolved unless explicitly set, so it cannot work with PnP. It should be done like here: https://github.com/WeiAnAn/jest/blob/301a5cda50bf82617fe776938ee3fbd1c24b4a7c/packages/jest-config/src/normalize.ts#L477-L479

Reply:
We resolve it in `normalize` already don't we?

Reply:
I'm checking, but I think it only get resolved when the `testSequence` option is explicitly set - otherwise it uses the default value as-is.